src/components/Card/Card.js

```diff
@@ -47,28 +71,57 @@ export const Card = (note) => {
     .splice(0, 4)
     .join(" ");
 
+  const selectNote = (note) => {
+    notesDispatch({
+      type: "SELECT_NOTE",
+      payload: { note: note, id: note.id },
+    });
+  };
+
   return (
     <div
+      onClick={() => {
+        if (location.pathname === "/notes") {
```

Comment:
Here, as the function is not 3 or 4 lines and it also contains logic, so it would be good if you will create a function for this, instead of directly writing here. It will increase the code readability.

Reply:
That's a good idea. Thank you for the suggestion! I will make the changes.